skate-plugin/src/main/kotlin/com/slack/sgp/intellij/featureflags/FeatureFlagAnnotator.kt

```diff
@@ -58,7 +59,10 @@ class FeatureFlagAnnotator : ExternalAnnotator<List<FeatureFlagSymbol>, List<Fea
     }
   }
 
-  private fun isKotlinFeatureFile(psiFile: PsiFile): Boolean = psiFile.name.endsWith("Feature.kt")
+  private fun isKotlinFeatureFile(psiFile: PsiFile): Boolean {
+    val filePattern = psiFile.project.featureFlagFilePattern() ?: return false
+    return psiFile.name.matches(Regex(filePattern.trim()))
```

Comment:
You should use the pattern exactly as saved, not trimmed.

You can also tweak it to be filePattern.toRegex().matches(psiFile.name)

Reply:
Is there any harm in trim()? I want to ensure we don't break functionality for a user just because of an accidental space in the manual entry field

Reply:
Yes, a space is a valid regex character. You should avoid hidden modifications like this